src/components/pages/games/Tokens.jsx

```diff
@@ -78,6 +79,7 @@ const getTokenData = (game, tokens, paper) => {
     layout,
     perRow,
     perColumn,
+    perPageCalculations,
```

Comment:
Is there a reason to return this? I don't think we're want to use it over the `perPage` value. Unless there is a good reason I wouldn't return it.

Reply:
I don't think there is a really good reason, so I will just remove it from return statement.

Reply:
fixed [here](https://github.com/18xx-maker/18xx-maker/pull/646/commits/7b948eba3aa2ec9364111208840b4bec7976818b)